tests/Files.InteractionTests/SessionManager.cs

```diff
@@ -16,7 +16,8 @@ public class SessionManager
 		private const string WindowsApplicationDriverUrl = "http://127.0.0.1:4723";
 		private static string[] FilesAppIDs = new string[]{
 			"49306atecsolution.FilesUWP_dwm5abbcs5pn0!App",
-			"FilesDev_ykqwq8d6ps0ag!App"
+			"FilesDev_ykqwq8d6ps0ag!App",
```

Comment:
@chingucoding can we remove the old id if it's not longer being used?

Reply:
Yeah i think we can remove it

Reply:
Which one? I'm gonna remove and see test result

Reply:
The one in line 19 and maybe even the one in line 18

Reply:
@0x5bfa

Reply:
Removal of line 18 was caused to fail tests. Removed line 19 and will see result.

Reply:
What about the ones on 19 and 20?

Reply:
They were added in the PR in order to run properly.

Reply:
@chingucoding can you clarify what each one is for? It might help to also add comments in the code for this.

Reply:
Only the last one FilesDev_dwm5abbcs5pn0 is needed for the CI, the other ones were probably needed at some point for the CI or local development. I think adding comments for them would be very helpful.